Integrations/Whois/Whois.py

```diff
@@ -8245,8 +8245,10 @@ def get_whois(domain, normalized=None):
 
 
 def whois_command():
-
-    domain = demisto.args().get('query')
+    if demisto.args().get('query'):
+        domain = demisto.args().get('query')
+    else:
+        domain = demisto.args().get('domain')
```

Comment:
please move the context and hr to a create context and hr function, than create a `whois_command()` and a `domain_command()` and they will use the context and hr functions.
so you will not differentiate by checking the args.